php/class-photonfill-transform.php

```diff
@@ -111,14 +111,26 @@ public function transform_photon_url( $args, $image_url, $scheme = null ) {
 		 * Get the width and height of the default transform. Can be used for other transforms.
 		 */
 		public function get_dimensions( $args ) {
+			$width = ( ! empty( $args['width'] ) ) ? $args['width'] : 1000;
+			$height = ( ! empty( $args['height'] ) ) ? $args['height'] : 1000;
```

Comment:
* Why is this necessary?
* Why 1000 for these?
* Under what circumstances would they be empty, and are there any valid cases that this would then break?